Yoast/YoastSEO.js
# Set snippet preview default to desktop

## Summary

This PR can be summarized in the following changelog entry: 

* Changes the snippet preview default view from mobile to desktop.

## Relevant technical choices:

* Reverted all changes made in https://github.com/Yoast/YoastSEO.js/pull/1391

## Test instructions

This PR can be tested by following these steps:

* Check out this branch as a dependency in WordPress SEO by running `yarn add git+https://github.com/Yoast/yoastseo.js#stories/1441-set-snippet-preview-default-to-desktop`. (You might first have to run `yarn install`).
* Navigate to `node_modules/yoastseo` and run both `yarn`and `grunt build`.
* Navigate back to your `wordpress-seo` directory and run `grunt build`.
* Ensure that the Snippet Preview now displays the Desktop view as default and that there is no horizontal scrollbar present.
* Add a title and vary the length. Make sure that the `SEOTitleLength` assessment returns appropriate feedback. Please note that the issue described in https://github.com/Yoast/YoastSEO.js/issues/1196 is still present.

Fixes #1441 
Fixes https://github.com/Yoast/wordpress-seo/issues/9209
